src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -585,24 +588,47 @@ export const selectActiveQuoteAffiliateBps: Selector<ReduxState, string | undefi
     return activeQuote.affiliateBps
   })
 
-export const selectQuoteAffiliateFeeUserCurrency = createSelector(
-  selectActiveQuote,
-  selectQuoteSellAmountUserCurrency,
+type AffiliateFeesProps = {
+  feeModel: ParameterModel
+  inputAmountUsd: string | undefined
+}
+
+export const selectCalculatedFees: Selector<ReduxState, CalculateFeeBpsReturn> = createSelector(
+  [
+    (_state: ReduxState, { feeModel }: AffiliateFeesProps) => feeModel,
+    (_state: ReduxState, { inputAmountUsd }: AffiliateFeesProps) => inputAmountUsd,
+    (state: ReduxState, { feeModel }: AffiliateFeesProps) => selectVotingPower(state, { feeModel }),
+  ],
+  (feeModel, inputAmountUsd, votingPower) => {
+    const fees: CalculateFeeBpsReturn = calculateFees({
+      tradeAmountUsd: bnOrZero(inputAmountUsd),
+      foxHeld: votingPower !== undefined ? bn(votingPower) : undefined,
+      feeModel,
+    })
+
+    return fees
+  },
+)
+
+export const selectTradeAffiliateFeeAfterDiscountUsd = createSelector(
```

Comment:
we were using the `...Quote...` venacular to clarify its different to `...Input...` data

Reply:
since quote and input are both "trade"

Reply:
https://github.com/shapeshift/web/pull/6503/commits/956ea77d049d95450b99df1d890afb07c074087e